src/components/attendee/VideoHuddle/VideoHuddle.module.scss

```diff
@@ -8,31 +11,31 @@
   flex-direction: row-reverse;
 
   @media (min-width: 768px) {
-    @include sparkle-panel;
+    @include sparkle.panel--normal;
     grid-column: 1 / 3;
-    margin: 0 $whitespace * -1;
-    width: calc(100% + #{$whitespace * 2});
+    margin: 0 space.empty(-1);
```

Comment:
not sure how I feel about negative values for these functions ( especially with `empty` )
maybe something like `space.gap(-X)` would look better?